docs/nnue.md

```diff
@@ -2345,7 +2345,7 @@ if __name__ == '__main__':
 
 ### Using custom CUDA kernels
 
-How to run our own kernel? Don't we need a complicated setup with the CUDA compiler and all that? CuPy to the rescue. CuPy is a python library that allows easy creation of CUDA kernels using plain python strings containing the CUDA code. CuPy handles compilation and everything else for us. For example:
+How to run our own kernel? Don't we need a complicated setup with the CUDA compiler and all that? CuPy to the rescue. CuPy is a python library that allows the easy creation of CUDA kernels using plain python strings containing the CUDA code. CuPy handles compilation and everything else for us. For example:
```

Comment:
imo reads easier and is clearer without the "the"

Reply:
Reverted those ones.